f5-distributed-cloud/README.md

```diff
@@ -0,0 +1,99 @@
+## Overview
+
+F5 Distributed Cloud (XC) Services provides a global cloud native platform where customers can deploy, manage, and secure their applications regardless of whether the application resides in a public cloud, in a private data center, or a colocation facility (see below). It provides a variety of both ADN and CDN services.
+
+![snapshot][1]
+
+Although the F5 XC console UI provides very good observability natively, many enterprises prefer to aggregate their telemetry from various sources and centralize visibility/analytics on the Datadog platform.  To this end, the F5 XC platform includes the Global Log Receiver.
+
+A F5 XC global log receiver can be configured to securely send logs to a Datadog HTTPS logging endpoint. Configuration is quite simple and can be done via the [F5 XC Console UI][6].
+
+In addition to the following configuration guidance. The F5 Distributed Cloud Services integration includes:
+
+- Dashboard - *Access Log Overview*
+- Saved View - *Including facets for commonly queried fields*
+
+![snapshot][2]
+
+## Setup
+
+You can configure global log streaming in either system namespace or in shared namespace. In case of configuring in shared namespace, you can configure to stream from either shared namespace or all namespaces or specific list of namespaces. If you are configuring in system namespace, you can only stream logs from system namespace.
+
+The example shown in this guide creates a global log receiver object in the Console in system namespace for streaming the logs to the external log collection system.
+
+Perform the following in the F5® Distributed Cloud Console:
+
+**Step 1: Start creating a global log receiver**
+
+- In the Console home page, select Shared Configuration service.
+- Select Manage > Global Log Receiver.
+- Select Global Log Receiver in case of Cloud and Edge Sites service.
+
+![snapshot][3]
+
+- Select Add Global Log Receiver button.
+
+**Step 2: Configure global log receiver properties**
+Do the following in the Global Log Receiver section:
+
+- Enter a name in the metadata section. Optionally, set labels and add a description.
+- Select Request Logs or Security Events for the Log Type field. The request logs are set by default.
+- Select events to be streamed based on namespace from the following options:
+    - Select logs from current namespace - streams logs from the shared namespace.
+    - Select logs from all namespaces - streams logs from all namespaces.
+    - Select logs in specific namespaces - streams logs from specified namespaces. Enter the namespace name in the displayed namespaces list. Use Add item button to add more than one namespace.  Namespaces provide logical grouping and isolation of objects within a distributed cloud tenant.
+- Select Datadog for the Receiver Configuration box. Configure following for the Datadog receiver:
+    - Provide the appropriate data Doc site name, (datadoghq.com) in this case.   
+    - Provide a Datadog API key.
+
+![snapshot][4]
```

Comment:
please see comment regarding images in the overview section.

Reply:
updated images will be in next pull request